ui/lib/login.js

```diff
@@ -25,3 +25,35 @@ export function saveToVisitedOrgs(domain, orgName) {
     })
   }
 }
+
+export function currentBaseDomain() {
+  let domain = window.location.host
+  let parts = domain.split('.')
+  if (parts.length > 2) {
+    parts.shift() // remove the org
+    domain = parts.join('.') // join the last two parts of the domain
+  }
+  return domain
+}
+
+export function currentOrg() {
+  let domain = window.location.host
+  let parts = domain.split('.')
```

Comment:
we probably don't ned `let domain = window.location.host` we can just do 
`const parts = window.localhost.host.split('.')`

Reply:
that is nicer 👍